basic/src/tags/components/ColorRadioButtons.tsx

```diff
@@ -0,0 +1,61 @@
+import { ColorRadioButton } from "./ColorRadioButton";
+import { generateBrightColor } from "./colors";
+
+interface ColorRadioButtonsProps {
+  color: string;
+  setColor: (color: string) => void;
+}
+
+export function ColorRadioButtons({ color, setColor }: ColorRadioButtonsProps) {
+  const randomColor = generateBrightColor();
+
+  return (
+    <fieldset className="flex flex-col gap-2">
+      <legend className="text-black">Color:</legend>
```

Comment:
```suggestion
      <legend className="text-black">Color</legend>
```

Nitpick: I feel like `:` is not needed here.